src/pages/Admin/MapPreview/MapPreview.scss

```
@@ -0,0 +1,20 @@
+@import "scss/constants.scss";
+
+.MapPreview {
+  width: 100%;
+  margin-bottom: 1rem;
+  position: relative;
+  overflow: hidden;
+
+  &__save-button {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    margin-bottom: 0;
```

```suggestion
    margin: 0;
```
Removes right side space as well as bottom.

---

Fixed in: https://github.com/sparkletown/sparkle/pull/2006/commits/0c72b4d0e7a6245b75d442138529cfd256342c4b